pkg/network/netlink/circuit_breaker_test.go

```diff
@@ -0,0 +1,90 @@
+package netlink
```

Comment:
Can we add a test to sanity check that the math is right? 

Something like:

```
t := Ticker{} 
t.Tick(100)
t.Update(now)
t.Tick(10)
t.Update(now.Add(time.Second))

assert.Equal(t.eventRate, 28)
```

Reply:
This is done.